metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/CURRENT/package/files/bro_index.template

```diff
@@ -238,6 +238,538 @@
         "qtype_name": {
           "type": "string",
           "index": "not_analyzed"
+        },
+        "analyzer": {
+          "type": "string",
+          "index": "not_analyzed"
+        },
+        "failure_reason": {
```

Comment:
I'm sorry if this is left field but..... What if, we provide more than one 'example' template in the bro parser extension, and not try to come up with the perfect configuration?

Also - if and when 777 hits and we move on to 'create an instance of a parser - clone and edit configurations' they can use those samples to do so.